docs/source/tutorial/spaceflights_tutorial_faqs.md

```diff
@@ -0,0 +1,59 @@
+# Spaceflights tutorial FAQs
+
+
+## How do I resolve these common errors?
+
+### Data Catalog setup
+You're [testing whether Kedro can load the companies or reviews data](./set_up_data.md#test-that-kedro-can-load-the-csv-data) and see the following:
+
+```python
+DataSetError: Failed while loading data from data set
+CSVDataSet(filepath=...).
+[Errno 2] No such file or directory: '.../companies.csv'
+```
+
+Have you downloaded [the three sample data files](./set_up_data.md#download-datasets) and stored them in the `data/raw` folder?
+
+Or maybe you see the following:
+
+```python
+DataSetNotFoundError: DataSet 'companies' not found in the catalog
+```
+
+Make sure you have saved `catalog.yml`. Call `exit()` within the ipython session and restart `kedro ipython`. Then try again.
```

Comment:
I wonder would it be useful to keep the `ipython` terminal open throughout the tutorial, and use the `%reload_kedro` magic to load the new data instead.